gitzones/auth

```diff
@@ -64,10 +64,12 @@ class AuthorizedKeys(list):
         return self.db.copy()
 
     def _save(self) -> None:
-        with open(self.location + '~', "w") as f:
+        real_location = os.path.realpath(self.location)
+        with open(real_location + '~', "w") as f:
             for label, proto_key in self.db.items():
                 f.write(proto_key + " " + label + "\n")
-        os.rename(self.location + '~', self.location)
+        os.rename(real_location + '~', os.path.realpath(real_location))
```

Comment:
`os.path.realpath(real_location)` can be simply `real_location`, I think

Reply:
fixed